astropy/io/ascii/cds.py

```diff
@@ -173,7 +194,11 @@ def get_cols(self, lines):
                 cols.append(col)
             else:  # could be a continuation of the previous col's description
                 if cols:
-                    cols[-1].description += line.strip()
+                    cols[
+                        -1
+                    ].description += (
+                        f"{'' if cols[-1].description == '' else ' '}{line.strip()}"
+                    )
```

Comment:
```suggestion
                    cols[-1].description += '' if cols[-1].description == '' else ' '
                    cols[-1].description += line.strip()
```
The first form may be structurally more elegant, but its formatting is so horrible mangled by Black that I'd much rather prefer a slightly broken down version.